lib/spack/spack/cmd/help.py

```diff
@@ -22,16 +22,27 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+import sys
+
 description = "get help on spack and its commands"
+section = "help"
+level = "short"
 
 
 def setup_parser(subparser):
-    subparser.add_argument('help_command', nargs='?', default=None,
-                           help='command to get help on')
+    help_cmd_group = subparser.add_mutually_exclusive_group()
+    help_cmd_group.add_argument('help_command', nargs='?', default=None,
+                                help='command to get help on')
```

Comment:
I'm not sure I understand. Mutually exclusive groups generally have multiple options, only one of which can be used at the same time. Here you have 2 mutually exclusive groups. Did you mean to make one mutually exclusive group so that you can either run `spack help <command>` or `spack help --all` but not both?